pt-BR/mail/chrome/messenger/AccountWizard.dtd

```diff
@@ -23,26 +23,26 @@
 <!ENTITY fullnameLabel.label "Nome:">
 <!ENTITY fullnameLabel.accesskey "N">
 
-<!ENTITY emailLabel.label "Email:">
+<!ENTITY emailLabel.label "E-mail:">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY emailLabel.label "Correio eletrônico:">